lib/trino-parquet/src/main/java/io/trino/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -175,72 +228,65 @@ public static Domain getDomain(
         }
 
         if ((type.equals(BIGINT) || type.equals(TINYINT) || type.equals(SMALLINT) || type.equals(INTEGER)) && (statistics instanceof LongStatistics || statistics instanceof IntStatistics)) {
-            Optional<ParquetIntegerStatistics> parquetIntegerStatistics = toParquetIntegerStatistics(statistics, id, column);
-            if (parquetIntegerStatistics.isEmpty() || isStatisticsOverflow(type, parquetIntegerStatistics.get())) {
+            ParquetLongStatistics parquetIntegerStatistics = toParquetLongStatistics(statistics, id, columnName);
```

Comment:
The variable name no longer matches the type. Also, call it just `longStatistics`. No need for the overly verbose name.